msteams-platform/concepts/deploy-and-publish/appsource/prepare/Test-preview-for-monetized-apps.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Test preview for monetized apps 
+author: v-ypalikila
+description: Create and test SaaS Preview offers for Teams app before pushing the offer live.
+ms.topic: conceptual
+ms.author: v-ypalikila
+ms.localizationpriority: medium
+keywords: teams apps SaaS offer preview offer  app source preview
+---
+
+# Test preview for monetized apps
+
+As a developer, you can create a Software as a Service (SaaS) offer and test the end-to-end purchase experience of an offer in Teams before you publish the offer. Users who are added as the preview audience for the Teams app can test the preview offer, see [ Add a preview audience for a SaaS offer](/azure/marketplace/create-new-saas-offer-preview).
+
+## Configure your app with the Preview offer ID
+
+After adding the preview audience to your SaaS offer, link the offer ID to your Teams app for users to see your subscription plans in the Teams store.
+
+1. Go to [Developer Portal](https://dev.teams.microsoft.com/).
+1. Select **Apps** from the left pane.
+1. Select the app to link the SaaS offer to.
+1. Select **Plans and pricing** and enter the **Publisher ID** and **Offer ID**.
+  1. Add the *-Preview* suffix to the offer ID.
+1. Select **View** to preview your subscription plans.
+1. Review the plans listed under **Apps Subscription** and click **Save**.
+
+    :::image type="content" source="../../../../assets/images/saas-offer/Plans-and-pricing-preview-offer.png" alt-text="configure preview offer":::
+
+The subscriptionOffer property is added to your app manifest.
+
+```json
+"subscriptio(nOffer": {
+     "offerId": "publisherId.offerId"-preview  
+     }
+```
+>[!NOTE]
+> Check for the label *preview offer* next to **Apps Subscription** to confirm if the offer is a preview offer.
+
+## Sideload the app to Teams
+
+After configuring your app with the preview offer ID, create an updated app package and upload the app package to Teams for the preview audience to test purchasing the app. For more information, see [Upload your app in Microsoft Teams](/concepts/deploy-and-publish/apps-upload).
+ 
+You can select **Preview in Teams** in the developer portal to launch your app quickly in the Teams client.
+
+* If the preview offer and preview audience are specified in the app manifest, the user can see the **Buy a Subscription** button.
```

Comment:
The "preview audience" is not specified in app manifest. They are defined in Partner Center on the offer as below screenshot.
![image](https://user-images.githubusercontent.com/80368287/148858407-1ee81790-2b32-4e8f-b548-df8d0531c529.png)